### ISSUE TITLE
 XBB.2.3/DV.7.1 recombinant, with DV.7.1 Spike (5 seqs, Spain,Italy, Luxembourg)

### ISSUE THREAD

**Issue by corneliusroemer** [sars-cov-2-variants/lineage-proposals#674]
This Spanish sequence looks like an XBB.2.3/DV.7.1 recombinant with breakpoint exactly between 22017-22031 (both ends inclusive).

Seems to be missing from Usher tree due to QC? @AngieHinrichs

hCoV-19/Spain/IB-HUSE-08555/2023|EPI_ISL_18106845|2023-08-08

<img width="651" alt="image" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/25161793/0afc518a-1b9a-44d6-9a74-3e89e10b6205">

<img width="1492" alt="image" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/25161793/0e22bb9a-c89b-4289-a4aa-1ef81fe885ae">

Singlet when placed in Usher. It appears in a meaningless place (BA.4_dropout)
<img width="1561" alt="image" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/25161793/a7614b35-04c3-4b64-9a85-fdf166231cda">

**(EDITED):**
 Thanks to @Sinickle who has found [more samples](https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_25eac_742490.json?c=userOrOld&label=id:node_7046697) of this recombinant (maybe he wants to add more on that).
   

I found a new query that catches all 5 sequences: , **T13560C, T17661C, T25959C**

Samples IDs: EPI_ISL_18106845, EPI_ISL_18118213, EPI_ISL_18215936,
EPI_ISL_18218415, EPI_ISL_18220496

New Tree:
  
![Schermata 2023-09-05 alle 19 07 53](https://github.com/sars-cov-2-variants/lineage-proposals/assets/87669813/b5118b2d-0cb0-4497-b3d0-15abdc501c21)

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_3491d_75b310.json?label=id:node_10802729

**Comment by AngieHinrichs** [comment]
> Seems to be missing from Usher tree due to QC? @AngieHinrichs

Yep, it's filtered out for having too many reversions (6 > 5).  I will add an exception for it and it should be added in the 2023-08-25 build.

**Comment by AngieHinrichs** [comment]
If there are others like it (with >5 reversions relative to nextclade placement), they will be filtered out too until I add exceptions for them, so please let me know if more appear.

**Comment by aviczhl2** [comment]
+1 Italy EPI_ISL_18125936  @AngieHinrichs

**Comment by aviczhl2** [comment]
EPI_ISL_18118213 seems to have a terrible Spike?

**Comment by FedeGueli** [comment]
> EPI_ISL_18118213 seems to have a terrible Spike?

yeah. all fshifted

**Comment by FedeGueli** [comment]
4 out of 5 sequences seem to have Orf1a.T1822I (C5730T) = NSP3_T1004I = PLPro_T259I

**Comment by AngieHinrichs** [comment]
> Samples IDs: EPI_ISL_18106845, EPI_ISL_18118213, EPI_ISL_18215936, EPI_ISL_18218415, EPI_ISL_18220496

Thanks!  I added exceptions but today's build is already underway so they should go in tomorrow's build, 2023-09-06.

**Comment by Sinickle** [comment]
I don't have too much more to add -- 

I feel like this could quite plausibly be a more fit form of DV.7.1 -- S:F157L is a fitness improving mutation on XBB (ie, convergent and measured to improve growth rate advantage), so the breakpoint being between S:152 and S:157 could reasonably be one of the more advantageous possibilities. 

DV.7.1 still looks pretty strong in Asturias Spain which I believe has had its earliest/strongest presence. With this proposed recombinant being at 5 sequences with all recent sequences and 3 countries, I personally feel like this is one of the top variants to be keeping a watch on/should be designated. 

![image](https://github.com/sars-cov-2-variants/lineage-proposals/assets/94097102/ce543ee5-e65b-4581-9890-668d50264cc2)

**Comment by aviczhl2** [comment]
Will further seqs continue to be removed if we don't designate?

The 5-reversion rule starts counting from a designated lineage, right?

**Comment by corneliusroemer** [comment]
Just noticed it again when looking at uploads from today - might designate after thorough review. Thanks for the discussion here, everyone! Will try the query `T13560C, T17661C, T25959C`

**Comment by corneliusroemer** [comment]
We can actually narrow down the XBB.2.3 donor pretty well! All 5 private mutations on that side up to breakpoint on top of XBB.2.3 occurred in a few sequences from Spain/France!

This is the donor, XBB.2.3.13:
<img width="1424" alt="image" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/25161793/807768a9-10a3-4903-a075-d69cbdf70d3e">
<img width="1569" alt="image" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/25161793/bc582468-ef67-4b29-9023-79bb05e65293">

Donor presence in Spain matches with fact that the other donor DV.7.1 is most common in Spain, and that the first sequence is from Spain.

**Comment by FedeGueli** [comment]
> We can actually narrow down the XBB.2.3 donor pretty well! All 5 private mutations on that side up to breakpoint on top of XBB.2.3 occurred in a few sequences from Spain/France!
> 
> This is the donor, XBB.2.3.13: <img alt="image" width="1424" src="https://user-images.githubusercontent.com/25161793/265811042-807768a9-10a3-4903-a075-d69cbdf70d3e.png"> <img alt="image" width="1569" src="https://user-images.githubusercontent.com/25161793/265811049-bc582468-ef67-4b29-9023-79bb05e65293.png">
> 
> Donor presence in Spain matches with fact that the other donor DV.7.1 is most common in Spain, and that the first sequence is from Spain.

Grear catch! Didnt check it went from singlet to lineage quite fastly!

**Comment by corneliusroemer** [comment]
The other side can be narrowed down as well to this branch of DV.7.1:
<img width="1550" alt="image" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/25161793/5c34b609-00b7-42c0-9d5c-5426f449bc50">

<img width="1554" alt="image" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/25161793/c84f730e-9a03-4616-837b-96502a103488">
https://next.nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_e590_789090.json?c=country&gt=S.858I&label=id:node_7691732

**Comment by FedeGueli** [comment]
> The other side can be narrowed down as well to this branch of DV.7.1: <img alt="image" width="1550" src="https://user-images.githubusercontent.com/25161793/265812893-5c34b609-00b7-42c0-9d5c-5426f449bc50.png">
> 
> <img alt="image" width="1554" src="https://user-images.githubusercontent.com/25161793/265813355-c84f730e-9a03-4616-837b-96502a103488.png"> https://next.nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_e590_789090.json?label=id:node_7691732

This parental dv.7.1 branch was transferred to main page today i think

**Comment by AngieHinrichs** [comment]
> The 5-reversion rule starts counting from a designated lineage, right?

I use command-line nextclade's very detailed output for filters, so it starts from Omicron lineages that are recognized by nextclade.  Nextclade is pretty much always more up to date than pangolin and often includes lineages that have not yet been in a pango-designation release (unlike pangolin-data), but it may not necessarily have all of the most recently designated lineages (especially because I need to remember to update nextclade data when there is a release).

**Comment by Over-There-Is** [comment]
> > The other side can be narrowed down as well to this branch of DV.7.1: <img alt="image" width="1550" src="https://user-images.githubusercontent.com/25161793/265812893-5c34b609-00b7-42c0-9d5c-5426f449bc50.png">
> > <img alt="image" width="1554" src="https://user-images.githubusercontent.com/25161793/265813355-c84f730e-9a03-4616-837b-96502a103488.png"> https://next.nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_e590_789090.json?label=id:node_7691732
> 
> This parental dv.7.1 branch was transferred to main page today i think

Yes, it is cov-lineages/pango-designation#2258

**Comment by Sinickle** [comment]
A sequence in Finland yesterday and another in Spain today, meaning that the last 6 sequences (there's only 7 total) were uploaded in the last 5 days, in 5 different countries.

**Comment by FedeGueli** [comment]
> A sequence in Finland yesterday and another in Spain today, meaning that the last 6 sequences (there's only 7 total) were uploaded in the last 5 days, in 5 different countries.

def one to watch!

**Comment by JosetteSchoenma** [comment]
A new one from the Netherlands. EPI_ISL_18263413.

**Comment by aviczhl2** [comment]
It seems that additional XCM seqs are still removed even after designation.

**Comment by AngieHinrichs** [comment]
> It seems that additional XCM seqs are still removed even after designation.

Yep.  I will need to manually include sequences until the following things happen:

1. nextclade data for SARS-CoV-2 is updated to include XCM
2. I update nextclade (this will make sure new sequences aren't excluded)
3. I re-run nextclade on older sequences

So in the meantime if you could periodically send a list of names or IDs that would be super helpful!  I see EPI_ISL_18263413 and EPI_ISL_18262219 in Josette's note above and will add those now.  They should appear in the 2023-09-20 tree.

**Comment by aviczhl2** [comment]
> > It seems that additional XCM seqs are still removed even after designation.
> 
> Yep. I will need to manually include sequences until the following things happen:
> 
> 1. nextclade data for SARS-CoV-2 is updated to include XCM
> 2. I update nextclade (this will make sure new sequences aren't excluded)
> 3. I re-run nextclade on older sequences
> 
> So in the meantime if you could periodically send a list of names or IDs that would be super helpful! I see EPI_ISL_18263413 and EPI_ISL_18262219 in Josette's note above and will add those now. They should appear in the 2023-09-20 tree.

I see manual labels like BA.4-dropout of XBB.1.5_17124 existing on tree. Maybe a manual label is also needed for XCM?

**Comment by AngieHinrichs** [comment]
nextclade is the source of the number of reversions used for filtering, and the nextclade tree is built independently of the UShER tree.  Those manual labels are in the UShER tree, not in the nextclade tree.  They are for the benefit of Pango, i.e. when I make a minimized tree for use in pangolin, those labels help to identify the correct lineages of sequences that belong to a lineage but may have an odd placement in the UShER tree.

**Comment by aviczhl2** [comment]
It seems still buggy on XCM usher placement, most seqs are misread to have XBB.1 mutations Q183E, G252V, L368I, V445P,F490S. 


[usher](https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice43_genome_36559_8c96c0.json?c=gt-S_183,252,368,445,490&gmax=25384&gmin=21563)
![image](https://github.com/sars-cov-2-variants/lineage-proposals/assets/10445944/95f85901-003e-469b-bf8b-c621e96a068b)

**Comment by AngieHinrichs** [comment]
Yes, sorry, the accuracy of the tree's mutations for recombinants suffers because I apply branch-specific masking of common artifact reversions, so true reversions and mutations lost by recombination are ignored in the tree and it appears that they have mutations that they don't.

You can see the long list of positions (or specific reversions) that I mask out in the file [branchSpecificMask.yml](https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/utils/otto/sarscov2phylo/branchSpecificMask.yml), for example in [XBB (starting on line 137)](https://github.com/ucscGenomeBrowser/kent/blob/01089c81628328a3a9083680e1705238e4246472/src/hg/utils/otto/sarscov2phylo/branchSpecificMask.yml#L137) I mask out the reversions G22109C (S:183), A22664C (S:368), C22895G and C22896T (S:445),  and C23031T (S:490).  In [XBB.1](https://github.com/ucscGenomeBrowser/kent/blob/01089c81628328a3a9083680e1705238e4246472/src/hg/utils/otto/sarscov2phylo/branchSpecificMask.yml#L150) (> G27915T) I mask out the reversion of the XBB.1-defining T22317G (S:252).  

It's a trade-off -- if I don't mask reversions, then they can create a lot of false branching in the tree, so I mask when I see trouble.  But when I mask them, the tree has incorrect mutations for true reversions at those positions and for many recombinants.

**Comment by FedeGueli** [comment]
XCM

**Comment by JosetteSchoenma** [comment]
I am a bit lost on what is going on with the XCM Usher tree. Newly uploaded ones are on a branch with Spike reversions, while they are already there with mutations showing that they do not actually have, like S:F490S.
See this EPI_ISL_18403533, for example. But several from GenBank without an EPI_ISL number. @AngieHinrichs 
![image](https://github.com/sars-cov-2-variants/lineage-proposals/assets/94831035/9f5f40f1-ae74-4f23-b3b7-178ee2e11bb1)
![image](https://github.com/sars-cov-2-variants/lineage-proposals/assets/94831035/7bc002b0-feeb-4e61-a010-aae4b3c09082)
![image](https://github.com/sars-cov-2-variants/lineage-proposals/assets/94831035/e732843d-d005-4b85-b27e-4c1c1410dd79)
These are the 62 I get with my query. 9 new Dutch ones today.
EPI_ISL_18106845
EPI_ISL_18118213
EPI_ISL_18215936
EPI_ISL_18218415
EPI_ISL_18220496
EPI_ISL_18228332
EPI_ISL_18234105
EPI_ISL_18234460
EPI_ISL_18262219
EPI_ISL_18263413
EPI_ISL_18273371
EPI_ISL_18290110
EPI_ISL_18290697
EPI_ISL_18295453
EPI_ISL_18313581
EPI_ISL_18313714
EPI_ISL_18315600
EPI_ISL_18329964
EPI_ISL_18329980
EPI_ISL_18331311
EPI_ISL_18331338
EPI_ISL_18331345
EPI_ISL_18331346
EPI_ISL_18331735
EPI_ISL_18338530
EPI_ISL_18356463
EPI_ISL_18362762
EPI_ISL_18363960
EPI_ISL_18366537
EPI_ISL_18367143
EPI_ISL_18367151
EPI_ISL_18370443
EPI_ISL_18375963
EPI_ISL_18377320
EPI_ISL_18378282
EPI_ISL_18378287
EPI_ISL_18378294
EPI_ISL_18378311
EPI_ISL_18378825
EPI_ISL_18383761
EPI_ISL_18386740
EPI_ISL_18398287
EPI_ISL_18398491
EPI_ISL_18400049
EPI_ISL_18403514
EPI_ISL_18403521
EPI_ISL_18403533
EPI_ISL_18406416
EPI_ISL_18406423
EPI_ISL_18406640
EPI_ISL_18406656
EPI_ISL_18407796
EPI_ISL_18413802
EPI_ISL_18419891
EPI_ISL_18419892
EPI_ISL_18419893
EPI_ISL_18419899
EPI_ISL_18419904
EPI_ISL_18419994
EPI_ISL_18420013
EPI_ISL_18420034
EPI_ISL_18420052

**Comment by AngieHinrichs** [comment]
> I am a bit lost on what is going on with the XCM Usher tree. Newly uploaded ones are on a branch with Spike reversions, while they are already there with mutations showing that they do not actually have, like S:F490S.

That is an unfortunate side effect of branch-specific masking, sorry.  XBB has T23031C (S:F490S), but many sequences in XBB have erroneous reversions on 23031 (and many other Spike mutations).  So on the XBB branch, I mask out all C23031T reversions (and many other specific reversions, see [branchSpecificMask.yml](https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/utils/otto/sarscov2phylo/branchSpecificMask.yml#L136)).  Unfortunately when there are true reversions, or recombinants that don't have specific mutations because they have a different parental lineage at that position vs. the lineage where UShER placed the sequence, the UShER tree falsely reports the mutations.  It's a tradeoff -- in order to prevent a lot of bad branches that would be caused by sequences with false reversions, we allow some false mutations in recombinants (and rare true reversions).

**Comment by JosetteSchoenma** [comment]
> > I am a bit lost on what is going on with the XCM Usher tree. Newly uploaded ones are on a branch with Spike reversions, while they are already there with mutations showing that they do not actually have, like S:F490S.
> 
> That is an unfortunate side effect of branch-specific masking, sorry.  XBB has T23031C (S:F490S), but many sequences in XBB have erroneous reversions on 23031 (and many other Spike mutations).  So on the XBB branch, I mask out all C23031T reversions (and many other specific reversions, see [branchSpecificMask.yml](https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/utils/otto/sarscov2phylo/branchSpecificMask.yml#L136)).  Unfortunately when there are true reversions, or recombinants that don't have specific mutations because they have a different parental lineage at that position vs. the lineage where UShER placed the sequence, the UShER tree falsely reports the mutations.  It's a tradeoff -- in order to prevent a lot of bad branches that would be caused by sequences with false reversions, we allow some false mutations in recombinants (and rare true reversions).

Oke. Thanks for explaining. Is this also why there are samples on the XCM tree, while they probably are not XCM, since they do have a F490S mutation?

**Comment by AngieHinrichs** [comment]
> Is this also why there are samples on the XCM tree, while they probably are not XCM, since they do have a F490S mutation?

Hmmm.  It is a little suspicious if that is the only way in which they differ.  How sure are we about the samples that do vs. don't have that mutation -- are only consensus genome sequences available (like in GISAID or GenBank), or are any of them in SRA (with raw reads, viewable using Theo Sanderson's https://deeperseq.genomium.org/) I wonder?

**Comment by JosetteSchoenma** [comment]
> > Is this also why there are samples on the XCM tree, while they probably are not XCM, since they do have a F490S mutation?
> 
> Hmmm. It is a little suspicious if that is the only way in which they differ. How sure are we about the samples that do vs. don't have that mutation -- are only consensus genome sequences available (like in GISAID or GenBank), or are any of them in SRA (with raw reads, viewable using Theo Sanderson's https://deeperseq.genomium.org/) I wonder?

My mistake. They  probably do not actually have an F490S, Usher is only showing it. I read your comment too quickly this morning and forgot about my own.
So, if I understand this correctly now, Usher adds this F490S, because it assumes it is an error that it was missing. And in freshly uploaded ones, Usher has not added it yet, so it places the sample in the tree with a 490 reversion (and other reversions), while actually all XCM do not have F490S.

**Comment by AngieHinrichs** [comment]
It's a little complicated but here's how it works:
1. UShER places a new sequence.  It may or may not have S:F490S.  If it does not have S:F490S, but is placed in XBB, then yes it will have a reversion after placement.  But...
2. After running UShER to add new sequences, branch-specific masking is applied.  Any sequences that have been placed in XBB with a reversion on S:F490S will have that reversion erased.  (If the reversion was a false reversion as I believe is true in most cases, that's a good thing; if it's a real reversion then whoops, I've made it incorrect.)

So yes, a sequence without S:F490S is temporarily placed with a reversion -- but you never see that intermediate tree, because I do branch-specific masking (and then optimization to see if we can move branches around to make the tree even better after all new sequences have been placed, and then filtering to remove implausibly long branches) before updating the tree in usher.bio.  

In order to tell if a genome really does or doesn't have a mutation, it's best to look at the raw reads.  Then one can get a better idea of whether there is enough read coverage at that position to be confident about the base value, and if there are enough reads, then what base value(s) they have at that position.  It can still be pretty ambiguous, possibly implying a mixed infection, contamination, or other sequencing issues.

Bottom line: especially for recombinants, don't 100% trust the UShER tree to tell what mutations a sequence has or does not have -- there is a lot of masking and imputation of missing data.  In order to be sure, it's necessary to look at the genome sequence.  If the sequence has a GenBank accession, Theo's https://gensplore.theo.io/ is useful.  If raw data for the sequence has been submitted to SRA, then Theo's https://deeperseq.genomium.org/ can show the full detail.  If you're working with sequences downloaded from GISAID then [nextclade](https://clades.nextstrain.org/) is the way to go (or the GISAID details page's list of mutations for each sequence).

**Comment by aviczhl2** [comment]
> It's a little complicated but here's how it works:
> 
> 1. UShER places a new sequence.  It may or may not have S:F490S.  If it does not have S:F490S, but is placed in XBB, then yes it will have a reversion after placement.  But...
> 2. After running UShER to add new sequences, branch-specific masking is applied.  Any sequences that have been placed in XBB with a reversion on S:F490S will have that reversion erased.  (If the reversion was a false reversion as I believe is true in most cases, that's a good thing; if it's a real reversion then whoops, I've made it incorrect.)
> 
> So yes, a sequence without S:F490S is temporarily placed with a reversion -- but you never see that intermediate tree, because I do branch-specific masking (and then optimization to see if we can move branches around to make the tree even better after all new sequences have been placed, and then filtering to remove implausibly long branches) before updating the tree in usher.bio.
> 
> In order to tell if a genome really does or doesn't have a mutation, it's best to look at the raw reads. Then one can get a better idea of whether there is enough read coverage at that position to be confident about the base value, and if there are enough reads, then what base value(s) they have at that position. It can still be pretty ambiguous, possibly implying a mixed infection, contamination, or other sequencing issues.
> 
> Bottom line: especially for recombinants, don't 100% trust the UShER tree to tell what mutations a sequence has or does not have -- there is a lot of masking and imputation of missing data. In order to be sure, it's necessary to look at the genome sequence. If the sequence has a GenBank accession, Theo's https://gensplore.theo.io/ is useful. If raw data for the sequence has been submitted to SRA, then Theo's https://deeperseq.genomium.org/ can show the full detail. If you're working with sequences downloaded from GISAID then [nextclade](https://clades.nextstrain.org/) is the way to go (or the GISAID details page's list of mutations for each sequence).

Maybe shall do branch-specific unmasking of these positions for XCM(and XCT and future XBB/DV.7.1 recombs)?

**Comment by JosetteSchoenma** [comment]
@AngieHinrichs 
Since you say: 
'So yes, a sequence without S:F490S is temporarily placed with a reversion -- but you never see that intermediate tree, because I do branch-specific masking (and then optimization to see if we can move branches around to make the tree even better after all new sequences have been placed, and then filtering to remove implausibly long branches) before updating the tree in usher.bio.'

Does that mean that you did not do the branch-specific masking yet for XCM? Because there are many XCM with F490S on that tree and if I do a GISAID search for XCM with the query and then add F490S to the query afterwards, there are 0 samples left. So XCM really does not have F490S. And I guess that is the same for the other Spike mutations that show as reverted.

I think this is what @aviczhl2 means as well.

**Comment by AngieHinrichs** [comment]
> Does that mean that you did not do the branch-specific masking yet for XCM? Because there are many XCM with F490S on that tree and if I do a GISAID search for XCM with the query and then add F490S to the query afterwards, there are 0 samples left. So XCM really does not have F490S.

XCM sequences do not have F490S, but XCM in the tree falsely appears to have F490S -- that is caused by branch-specific masking in the XBB branch where XCM is placed.  Branch-specific masking is applied every day after UShER places new sequences in the tree, and you are seeing its undesirable effect on XCM.  Branch-specific masking prevents a lot of bad-branch structural problems in XBB as a whole, but it can cause mutation accuracy problems for recombinants and rare cases of true reversions.

**Comment by JosetteSchoenma** [comment]
Thanks for your elaborate answers, @AngieHinrichs . I certainly am not wishing to add to your workload. Simply trying to understand how this all works. I can indeed imagine your workload is more then big enough. 🙏

Would it maybe be a better idea to put XCM on the BA.2.75/DV.7.1 branch than on the XBB branch? Or would that simply cause the same problem for other mutations?

**Comment by AngieHinrichs** [comment]
> Would it maybe be a better idea to put XCM on the BA.2.75/DV.7.1 branch than on the XBB branch?

Maybe it would, but usher (and matOptimize) will place a branch wherever it has the fewest additional mutations.  So even if I could somehow force it to move to the other parent branch, matOptimize would probably move it back the next day.

> Or would that simply cause the same problem for other mutations?

Yes, that is also possible.
